docker-compose.yml

```diff
@@ -8,10 +8,26 @@ services:
       - ./:/usr/local/src/bash-commons
     working_dir: /usr/local/src/bash-commons/.circleci
     command: ./shellcheck.sh
-  bats:
+  bats_ubuntu1604:
     build:
       context: ./
-      dockerfile: Dockerfile.bats
+      dockerfile: Dockerfile.ubuntu16.04.bats
+    volumes:
+      # Mount all the files so you have "hot reload" of all changes from the host
+      - ./:/usr/local/src/bash-commons
+    working_dir: /usr/local/src/bash-commons
+    command: bats test
+    # Necessary so we can run a mock EC2 metadata service on port 80 on a special IP
+    privileged: true
+    ports:
+      # For moto
+      - "5000:5000"
+      # For ec2-metadata-mock
+      - "8111:8111"
```

Comment:
Are these the ports that conflict? Is it possible to change which ports are used for each OS?

Reply:
After coming back to this a month later, I realized these exposed ports were unnecessary because all the tests are running in docker. So I removed them.